docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>This is SO good!! :D</p><p> </p><p>Please add <code>plt.show()</code>  at the end of the cell to prevent <code>&lt;matplotlib.legend.Legend at 0x7fee05801f10&gt;</code>  from being displayed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>